src/unix/stream.c

```diff
@@ -973,8 +973,20 @@ uv_handle_type uv__handle_type(int fd) {
   memset(&ss, 0, sizeof(ss));
   sslen = sizeof(ss);
 
-  if (getsockname(fd, (struct sockaddr*)&ss, &sslen))
+  if (getsockname(fd, (struct sockaddr*)&ss, &sslen)) {
+#if defined(_AIX)
+    // On aix, ibmi, receiving RST from TCP instead of FIN immediately puts
+    // fd into an error state, despite some readable bytes of data remaining.
```

Comment:
```suggestion
    // fd into an error state.
```

That handling behavior is the TCP spec for handling RST, IIRC, so sounds odd to call it out specifically.